README.md

```diff
@@ -14,20 +14,58 @@ Read how this project is [governed](GOVERNANCE.md).
 
 Chat on [chat.mozilla.org#mdn](https://chat.mozilla.org/#/room/#mdn:mozilla.org).
 
-## Installation
+## Installation and Import
+
+### NodeJS
 
 You can install `@mdn/browser-compat-data` as a node package.
 
 ```
 npm install @mdn/browser-compat-data
+...or...
+yarn add @mdn/browser-compat-data
 ```
 
-## Usage
+Then, you can import BCD into your project with either `import` or `require()`:
 
 ```js
+// ESM w/ Import Assertions (NodeJS 16+)
```

Comment:
Might be better to avoid the `w/` abbreviation, which is not necessarily commonly known in non-English speaking countries like Germany:

```suggestion
// ESM with Import Assertions (NodeJS 16+)
```